### PR TITLE
dspdfviewer: fix build on Linux and use poppler-qt5

### PR DESCRIPTION
**Pull request by danielnachun**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/dspdfviewer.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class Dspdfviewer < Formula
   url "https://github.com/dannyedel/dspdfviewer/archive/v1.15.1.tar.gz"
   sha256 "c5b6f8c93d732e65a27810286d49a4b1c6f777d725e26a207b14f6b792307b03"
   license "GPL-2.0"
```
Comment (branchvincent) [inline Formula/dspdfviewer.rb:6]: https://github.com/dannyedel/dspdfviewer/blob/555d28b0ec31eb825e939d1b194eaf796d1f8c11/adjustedlink.cpp#L5-L8
```suggestion
  license "GPL-2.0-or-later"
```

**Comment by branchvincent** [comment]
```
==> /home/linuxbrew/.linuxbrew/Cellar/dspdfviewer/1.15.1_10/bin/dspdfviewer --help
qt.qpa.xcb: could not connect to display 
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, wayland-egl, wayland, wayland-xcomposite-egl, wayland-xcomposite-glx, webgl, xcb.
```

**Comment by danielnachun** [comment]
The test is now fixed.  I'm happy to take any suggestions for making it look better.

**Comment by danielnachun** [comment]
@branchvincent can you take a look again now?

File: Formula/dspdfviewer.rb [hunk 2]
```diff
@@ -27,65 +27,32 @@ class Dspdfviewer < Formula
...
 
   test do
-    system bin/"dspdfviewer", "--help"
+    on_macos do
+      system "#{bin}/dspdfviewer", "--help"
+    end
+    on_linux do
+      assert_match("qt.qpa.xcb: could not connect to display",
+                   shell_output("#{bin}/dspdfviewer --help 2>&1", 134))
+    end
```
Comment (branchvincent) [inline Formula/dspdfviewer.rb:56]: after digging, it looks like the linux error is due to our docker container lacking X11? https://github.com/therecipe/qt/issues/775#issuecomment-456899597

not sure if there's a more direct solution, but this is one workaround: 
```suggestion
    ENV["QT_QPA_PLATFORM"] = "minimal" if OS.linux?
    system bin/"dspdfviewer", "--help"
```

**Review by branchv — APPROVED** [review]

**Comment by danielnachun** [comment]
Thanks for this suggesetion @branchvincent.  There are some other formulae which should probably also use this instead of disabling the test on Linux, so I'm glad we now know about setting `QT_QPA_PLATFORM` to `minimal`.

**Pull request merged**